core/commands/partials/vagrant_configuration_generator.rb

```diff
@@ -113,6 +113,9 @@ def get_libvirt_definition(_cookbook_path, path, node_params)
           qemu.cpu_mode = 'host-passthrough'
           qemu.cpus = <%= vm_cpu %>
           qemu.memory = <%= vm_mem %>
+        <% if box == 'generic/opensuse15' %>
```

Comment:
The check MUST be performed for the availability of the key in configuration, not for the image type. In order to support this configuration for other boxes we must be able to simply modify the configuration.